core/field_image.ts

```diff
@@ -133,10 +129,12 @@ export class FieldImage extends Field {
    * Configure the field based on the given map of options.
    * @param config A map of options to configure the field based on.
    */
-  protected override configure_(config: AnyDuringMigration) {
+  protected override configure_(config: Config) {
     super.configure_(config);
-    this.flipRtl_ = !!config['flipRtl'];
-    this.altText_ = parsing.replaceMessageReferences(config['alt']) || '';
+    if (config.flipRtl) this.flipRtl_ = config.flipRtl;
+    if (config.alt) {
+      this.altText_ = parsing.replaceMessageReferences(config.alt);
+    }
```

Comment:
Even though `altText_` is set to `''` by default, this would lead to a change in functionality if `configure_` was ever called multiple times. (Maybe it's only ever intended to be called once, though, so that would only happen if a class that extends `FieldImage` improperly calls `configure_` more than once with differt configs.) If it *is* valid for `configure_` to be called multiple times, though, then:

```ts
const config1 = { alt: "abc" };
const config2 = {};
... .configure_(config1); // Sets alt to 'abc'
... .configure_(config2); // Previously set alt to '', though it remains 'abc' now
```

If it's valid to call `configure_` multiple times, the boolean value `flipRtl` would also never allow setting `this.flipRtl_` to false.

Reply:
Thanks for catching that! Luckily calling `configure_` multiple times is not a supported use case (really it's just supposed to be treated as part of the constructor, but it's been factored out because... I can't remember why, maybe JSON initialization). So I think we're good.